bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java

```diff
@@ -608,22 +611,18 @@ public void stopAutomaticRefresh() {
      */
     private void enableDHCPListen(InetAddress destinationAddress) {
         try {
-            if (DHCPListenService.register(destinationAddress.getHostAddress(), this).isUseUnprevilegedPort()) {
-                dhcpState = "No access right for port 67. Bound to port 6767 instead. Port forwarding necessary!";
-            } else {
-                dhcpState = "Running normally";
-            }
+            DHCPPacketListenerServer listener = DHCPListenService.register(destinationAddress.getHostAddress(), this);
+            dhcpState = String.format("Bound to port %d - %s", listener.getCurrentPort(),
+                    (listener.usingPrivilegedPort() ? "Running normally" : "Port forwarding necessary !"));
         } catch (SocketException e) {
-            logger.warn("Cannot use DHCP sniffing.", e);
+            dhcpState = String.format("Cannot use DHCP sniffing: %s", e.getLocalizedMessage());
+            logger.warn("{}", dhcpState);
```

Comment:
Why getLocalizedMessage ?
You are potentially mixing a text in English and a translated text.
And for logging, the recommendation is to remain in English.
Translation is only for what can be displayed in UI.

Reply:
Sorry for the delay @lolodomo , I was working on the netatmo binding that absorbed my time.